azure-pipelines.yml

```diff
@@ -65,7 +65,7 @@ stages:
                 cd SyntheticsRunTestsTask
                 yarn install --immutable
                 yarn build
-                yarn install --production --immutable
+                yarn workspaces focus --production
```

Comment:
Why use `workspaces` here? Is there really no more way to `install --production`?

Reply:
Yes, using `workspaces` is required now.

From the [Yarn 1.x to 2.x Migration Guide](https://yarnpkg.com/getting-started/migration#renamed): 

> Yarn Classic (1.x) | Yarn (2.x) | Notes
> -- | -- | --
> `yarn install --production` | `yarn workspaces focus --all --production` | Requires the `workspace-tools` plugin

And since we have a single workspace, `--all` is not needed.